hooks/useDogs.ts

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import { DogDetails } from "../context";
+import { breedsData } from "../data/breedsData";
+import { useLocalStorage } from "./useLocalStorage";
+import { getRandomArrayIndex } from "../utils/getRandomArrayIndex";
+
+type FetchedDog = {
+  message: string;
+  status: string;
+};
+
+type UseDogs = {
+  dogs: DogDetails[];
+  loading: boolean;
+  error?: Error;
+  loadMore: (howMany?: number) => Promise<void>;
+  setEmpty: () => void;
+};
+
+export const useDogs = (initalHowMany = 10): UseDogs => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<Error>();
+  const [dogs, setDogs] = useState<DogDetails[]>([]);
+
+  const [storagedBreeds] = useLocalStorage<string[]>(
+    "selectedBreeds",
+    breedsData
+  );
+
+  const loadMore = async (howMany = 4) => {
+    setLoading(true);
+
+    const getDog = async () => {
+      const randomIndex = getRandomArrayIndex(storagedBreeds);
+      const breedName = storagedBreeds[randomIndex];
+
+      const response = await fetch(
+        `${process.env.NEXT_PUBLIC_DOGS_API_PREFIX}${breedName}${process.env.NEXT_PUBLIC_DOGS_API_SUFFIX}`
```

Comment:
> process.env.NEXT_PUBLIC_DOGS_API_SUFFIX

I don't think it should be in envs

Reply:
I guess you meant that?
![image](https://user-images.githubusercontent.com/47990852/123751622-33d34d00-d8b8-11eb-9621-2620b2e9e04c.png)

Reply:
Yes